core/blockchain.go

```diff
@@ -134,6 +135,7 @@ type CacheConfig struct {
 	SnapshotLimit       int           // Memory allowance (MB) to use for caching snapshot entries in memory
 	Preimages           bool          // Whether to store preimage of trie key to the disk
 	TraceCacheLimit     int
+	MPTWitness          int // How to generate witness data for mpt circuit, 0: nothing, 1: natural
```

Comment:
make use of iota

Reply:
it means we should use a string type here?

Reply:
for example

```
type MPTWitnessType uint

const (
	MPTWitnessNothing MPTWitnessType = iota
	MPTWitnessNatural
)
```

Then `MPTWitnessNothing` will be `0` (in terms of value), `MPTWitnessNatural will be `1`

Reply:
Then line 152
````
	MPTWitness:      0,
```

can be 
````
	MPTWitness:      MPTWitnessNothing,
```